runtime/common/src/evm/mod.rs

```diff
@@ -107,3 +107,76 @@ impl<T: pallet_aura::Config<AuthorityId = AuraId>> FindAuthor<H160> for FindAuth
+
+/// Input for the KeccakHasher to derive a random `H160` where the passthrough
+/// router is always located at. Refers to address:
+/// `0x33e7daf228e7613ba85ef6c3647dbceb0f011f7c`
+const PASSTHROUGH_ROUTER_ACCOUNT_CODES_ACCOUNT_LOCATION_SALT: &[u8] =
+	b"PASSTHROUGH_ROUTER_ACCOUNT_CODES_ACCOUNT_LOCATION_SALT";
+
+pub fn passthrough_router_location() -> H160 {
+	H160::from(sp_core::KeccakHasher::hash(
+		PASSTHROUGH_ROUTER_ACCOUNT_CODES_ACCOUNT_LOCATION_SALT,
+	))
+}
+
+pub fn passthrough_genesis() -> (H160, fp_evm::GenesisAccount) {
+	(
+		passthrough_router_location(),
+		fp_evm::GenesisAccount {
+			nonce: Default::default(),
+			balance: Default::default(),
+			storage: Default::default(),
+			code: PASSTHROUGH_ROUTER_ACCOUNT_CODES.to_vec(),
+		},
+	)
+}
+
+#[cfg(test)]
+mod tests {
+	use sp_runtime::traits::{BlakeTwo256, Hash};
+
+	use super::*;
+
+	#[test]
+	fn stable_passthrough_location() {
+		assert_eq!(
+			passthrough_router_location().as_bytes(),
+			hex_literal::hex!("33e7daf228e7613ba85ef6c3647dbceb0f011f7c")
+		);
+	}
+
+	#[test]
+	fn stable_passthrough_bytecode_hash() {
+		assert_eq!(
+			BlakeTwo256::hash_of(&PASSTHROUGH_ROUTER_ACCOUNT_CODES.to_vec()),
+			hex_literal::hex!("545a48d6f7f1a01cb2bd090a5272cd52c54eafea762071ec652c8ac94610146e")
```

Comment:
@mustermeiszer This test fails. I suppose due to changes to the `PassThroughAdapter.sol` between your referenced commit `ee35da785f2af3303ef9f52c6ecefeb9671aa61b` and current main at `223a0f36edabc675f8c74c47b20e366178df7ca3`, see [diff](https://github.com/centrifuge/liquidity-pools/compare/ee35da785f2af3303ef9f52c6ecefeb9671aa61b). 

I wonder how you derived the passthrough account byte codes.

Error
```
thread 'evm::tests::stable_passthrough_bytecode_hash' panicked at runtime/common/src/evm/mod.rs:160:9:
assertion `left == right` failed
  left: 0x31173f15567854cfc3702aa6b639bf0dedf74638e745a3e90fa00f1619d8b94c
 right: 0x545a48d6f7f1a01cb2bd090a5272cd52c54eafea762071ec652c8ac94610146e
```

Reply:
As I can not compile it wanted to retrieve the new hash through the UI. Haha. So using the hash here generated will be okay

Reply:
But how can I derive the `PASSTHROUGH_ROUTER_ACCOUNT_CODES`?

Reply:
But then I only adjust the blake2 256 hash of the `PASSTHROUGH_ROUTER_ACCOUNT_CODES` but not any of the 3289 bytes of `PASSTHROUGH_ROUTER_ACCOUNT_CODES`. Based on your comments, those need to be adjusted as well?

```
/// Passthrough router deployed bytecode as of this state https://github.com/centrifuge/liquidity-pools/blob/ee35da785f2af3303ef9f52c6ecefeb9671aa61b/test/integration/PassthroughAdapter.sol
///
/// NOTE: If the above file changes, this code needs to be adapted.
...
```

Reply:
Does this change suffice @mustermeiszer ? [949bdff](https://github.com/centrifuge/centrifuge-chain/pull/1888/commits/949bdff2fbd5340560262d6f9343e780cc354573)

Reply:
Yes, that was, what I intended to do! Perfect! We always need that hash for setting up the routers, so I wanted to have it somewhere ^^